lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -409,6 +411,22 @@ function dosomething_user_validate_address_field($form, &$form_state) {
 }
 
 /**
+ * Validates that password is 6 or more characters long.  
+ *
+ * @param array $form
+ *  A drupal form.
+ * @param array $form_state
+ *  A drupal form_state array.
+ *
+ */
+function dosomething_user_validate_password_length($form, &$form_state) {
```

Comment:
in that case, we probably want this running on both `user_profile_form` and `user_register_form`